api/v1alpha1/openstack_ansibleee_types.go

```diff
@@ -28,7 +28,7 @@ const (
 	// Container image fall-back defaults
 
 	// OpenStackAnsibleEEContainerImage is the fall-back container image for OpenStackAnsibleEE
-	OpenStackAnsibleEEContainerImage = "quay.io/openstack-k8s-operators/openstack-ansibleee-runner:latest"
+	OpenStackAnsibleEEContainerImage = "quay.io/openstack-k8s-operators/openstack-ansibleee-runner:v0.1.0"
```

Comment:
this should be overriden in the bundel you should not need to ever change this form latest on main
is the AEE operator missing the ENV lookup

Reply:
I the bundle build pinning this properly then I think keeping the :latest here cannot (should not) break the dev preview build even if we land backaward incompatible changes in edpm_ansible right after the dev preview.

Reply:
This makes the most sense, thanks. We do have the env var setting the image url. We don't want to have to change the constant value on ever tagged release from a branch, so I agree we can just leave this latest.

Reply:
I'll close this PR then